src/ValueHandler/ChannelPricingValueHandler.php

```diff
@@ -28,16 +30,36 @@ final class ChannelPricingValueHandler implements ValueHandlerInterface
     /** @var string */
     private $akeneoAttribute;
 
+    /** @var string */
+    private $syliusPropertyPath;
+
+    /** @var PropertyAccessorInterface */
+    private $propertyAccessor;
+
     public function __construct(
         FactoryInterface $channelPricingFactory,
         ChannelRepositoryInterface $channelRepository,
         RepositoryInterface $currencyRepository,
-        string $akeneoAttribute
+        PropertyAccessorInterface $propertyAccessor = null,
+        string $akeneoAttribute,
+        string $syliusPropertyPath = 'price'
```

Comment:
> I don't think we could apply this feature without adding a Breaking Change...

Why you changed the sort order of the existing arguments? This is a BC. You can fix it with:

```suggestion
        string $akeneoAttribute,
        PropertyAccessorInterface $propertyAccessor = null,
        string $syliusPropertyPath = 'price'
```